plugin.rb

```diff
@@ -190,6 +190,20 @@ module ::PostVoting
     false
   end
 
+  register_modifier(:topic_embed_import_create_args) do |args|
+    category_id = args[:category]
+    return args unless category_id
```

Comment:
I kinda thought these `returns` had to be `next`s

Reply:
Good catch